### ISSUE TITLE
more graceful handling of retrieve of unknown ID

### ISSUE THREAD

**Issue by alaniwi** [ES-DOC/esdoc-errata-ws#9]
A `retrieve` of an unknown erratum id, for example:

https://errata.es-doc.org/1/issue/retrieve?uid=01234567-89ab-cdef-0123-456789abcdef

fails with an HTTP 500 status, with content:
```
{"errorCode": 999, "errorMessage": "'NoneType' object has no attribute 'to_dict'", "errorType": "AttributeError", "errorField": "--"}
```

It would probably be better to fail with a 400 ("bad request") in one form or another.

**Comment by AtefBN** [comment]
I agree Alan, I'll note this to be part of the errata coding sprint @momipsl and I will be holding end of this month.

**Comment by alaniwi** [comment]
@AtefBN Thanks.
